daffodil-tdml-processor/src/test/scala/org/apache/daffodil/tdml/TestTDMLRunner.scala

```diff
@@ -37,207 +37,6 @@ class TestTDMLRunner {
   val tns = example
   // val sub = XMLUtils.DFDL_XMLSCHEMASUBSET_NAMESPACE
 
-  @Test def testDocPart1() {
```

Comment:
FYI: This is where all the tests in the UnitTestTDMLRunner.scala file came from.